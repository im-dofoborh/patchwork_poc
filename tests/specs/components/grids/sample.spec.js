import { mount, createLocalVue } from '@vue/test-utils';
import SampleGrid from '~/components/grids/Sample';

describe('SampleGrid component', () => {
  let localVue = null;
  let wrapper = null;

  beforeEach(() => {
    localVue = createLocalVue();
    localVue.component('sample-grid', SampleGrid);
    wrapper = mount(SampleGrid, {
      localVue,
      slots: {
        default: '<div id="some-component" />',
      }
    })
  })

  it('renders default variant', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
