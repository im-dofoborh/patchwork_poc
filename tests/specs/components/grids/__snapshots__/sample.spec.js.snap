// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SampleGrid component renders default variant 1`] = `
<div class="sg-sample-grid">
  <div id="some-component"></div>
</div>
`;
